packages/reset/sanitize.css/assets.css

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
I guess we'd better exclude them from the source control.

Reply:
True. 🙂

Reply:
Have pushed a commit addressing your suggestion.